Cargo.toml

```diff
@@ -11,6 +11,7 @@ chrono = { version = "=0.4", features = ["clock"], default-features = false }
 displaydoc = "0.2"
 function_name = "0.3"
 loupe = "0.1"
+#massa-proto-rs = { git = "https://github.com/massalabs/massa-proto-rs.git", rev = "426fd325a55dfcc4033920bed2de075a7e7ad4b7" }
```

Comment:
what happened here ?

Reply:
update of massa-proto-rs : 

https://github.com/massalabs/massa-proto-rs/commits/main/